packages/mermaid/src/diagrams/sequence/sequenceDiagram.spec.js

```diff
@@ -2051,9 +2051,7 @@ describe('when rendering a sequenceDiagram with directives', () => {
 
   it('should handle one actor, when theme is dark and logLevel is 1 DX1 (dfg1)', async () => {
     const str = `
-%%{init: { "theme": "dark", "logLevel": 1 } }%%
 sequenceDiagram
-%%{wrap}%%
 participant Alice
 `;
     diagram = new Diagram(str);
```

Comment:
Sometimes tests address multiple things, then it can be good to split it up. Wrap for instance is actually tested in another test (in this case) so it is safe to remove.

Reply:
Wrap has nothing to do with `should handle one actor, when theme is dark and logLevel is 1 DX1 (dfg1)`, and was most likely an artefact from a copied test. No need to complicate the unit test as we have other places to test wrap as Knut pointed out.

Reply:
I've removed that block of tests as they are redundant now.